relayer/pkg/chainlink/ocr2/medianreport/report.go

```diff
@@ -59,7 +61,8 @@ func (c ReportCodec) BuildReport(oo []median.ParsedAttributedObservation) (types
 	var observations []junotypes.Felt
 	for i, o := range oo {
 		observers[i] = byte(o.Observer)
-		observations = append(observations, junotypes.BigToFelt(o.Value))
+		obs := starknet.BigIntToFelt(o.Value) // converst negative bigInts to corresponding felt in bigInt form
```

Comment:
Looks good, but the functions are now very similarly named: `starknet.BigIntToFelt`  and `junotypes.BigToFelt`. Maybe `SignedBigIntToFelt`? That's getting a bit long which is why I called the util helper `signedFelt`.

Reply:
renamed to `SignedBigToFelt` and `FeltToSignedBig` similar to the juno types but with `Signed`

also renamed additional utilities according to what @krebernisak suggested: https://github.com/smartcontractkit/chainlink-starknet/pull/81#discussion_r955150142